config_defaults.yml

```diff
@@ -588,11 +588,11 @@ Gemfile:
     ':system_tests':
       - gem: 'puppet-module-posix-system-r#{minor_version}'
         version: '~> 1.0'
-        condition: "Gem::Requirement.create(['>= 6.11.0', '< 8.0.0']).satisfied_by?(Gem::Version.new(puppet_version.dup))"
+        condition: "Gem::Requirement.create(['>= 6.11.0', '< 8.0.0']).satisfied_by?(Gem::Version.new(puppet_version.dup)) || ENV['PUPPET_GEM_VERSION'].nil?"
```

Comment:
please use `puppet_version` instead of `ENV['PUPPET_GEM_VERSION']`  - ref https://github.com/puppetlabs/pdk-templates/blob/f9bf5d8fd54ae45e2841915f1c477dad0221c893/moduleroot/Gemfile.erb#L61